build.gradle

```diff
@@ -20,6 +20,9 @@ repositories {
 		name = 'Fabric'
 		url = 'http://maven.modmuss50.me/'
 	}
+	maven {
+		url = 'https://jitpack.io'
```

Comment:
I'd like to wait until there's an official release published instead of using jitpack, since it can slow down our builds.

Reply:
They've pushed a build by now. Updated it.